### ISSUE TITLE
kindly asking for help: starting overtone

### ISSUE THREAD

**Issue by lwiechec** [overtone/overtone#482]
Hello, I tried to follow the docs but when I start the lein's REPL, I can see:

```
[luke:~/fun/overtone] $ lein repl
Retrieving org/clojure/clojure/1.2.0/clojure-1.2.0.jar from central
Could not find artifact overtone:overtone:jar:0.11 in central (https://repo1.maven.org/maven2/)
Could not find artifact overtone:overtone:jar:0.11 in clojars (https://repo.clojars.org/)
This could be due to a typo in :dependencies, file system permissions, or network issues.
If you are behind a proxy, try setting the 'http_proxy' environment variable.
Could not resolve dependencies
^C[luke:~/fun/overtone] 130 $ vi project.clj 
[luke:~/fun/overtone] $ lein repl
nREPL server started on port 41743 on host 127.0.0.1 - nrepl://127.0.0.1:41743
REPL-y 0.5.1, nREPL 0.8.3
Clojure 1.9.0-alpha17
Java HotSpot(TM) 64-Bit Server VM 1.8.0_291-b10
    Docs: (doc function-name-here)
          (find-doc "part-of-name-here")
  Source: (source function-name-here)
 Javadoc: (javadoc java-object-or-class-here)
    Exit: Control+D or (exit) or (quit)
 Results: Stored in vars *1, *2, *3, an exception in *e

user=> (use 'overtone.live)
--> Loading Overtone...
  ********************************************************** 
       WARNING: JVM argument TieredStopAtLevel=1 is active, and may 
       lead to reduced performance. This happens to currently be the 
       default lein setting:

       https://github.com/technomancy/leiningen/pull/1230

       If you didn't intend this JVM arg to be specified, you can turn 
       it off in your project.clj file or your global 
       ~/.lein/profiles.clj file by adding the key-val

       :jvm-opts ^:replace [] 
       ********************************************************** 
--> Booting internal SuperCollider server...
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
Cannot create RT messagebuffer thread: Operation not permitted (1)
Retrying messagebuffer thread without RT scheduling
Messagebuffer not realtime; consider enabling RT scheduling for user
no message buffer overruns
Cannot create RT messagebuffer thread: Operation not permitted (1)
Retrying messagebuffer thread without RT scheduling
Messagebuffer not realtime; consider enabling RT scheduling for user
no message buffer overruns
Cannot create RT messagebuffer thread: Operation not permitted (1)
Retrying messagebuffer thread without RT scheduling
Messagebuffer not realtime; consider enabling RT scheduling for user
no message buffer overruns
jackdmp 1.9.17
Copyright 2001-2005 Paul Davis and others.
Copyright 2004-2016 Grame.
Copyright 2016-2021 Filipe Coelho.
jackdmp comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details
JACK server starting in realtime mode with priority 10
self-connect-mode is "Don't restrict self connect requests"
audio_reservation_init
Acquire audio card Audio0
creating alsa driver ... hw:0|hw:0|1024|2|48000|0|0|nomon|swmeter|-|32bit
configuring for 48000Hz, period = 1024 frames (21.3 ms), buffer = 2 periods
ALSA: final selected sample format for capture: 32bit integer little-endian
ALSA: use 4 periods for capture
ALSA: Cannot open PCM device alsa_pcm for playback. Falling back to capture-only mode
Cannot use real-time scheduling (RR/10) (1: Operation not permitted)
AcquireSelfRealTime error
*** ERROR: dlsym load err '/home/luke/fun/overtone/native/linux/x86_64/libscsynth.so: undefined symbol: load'
Found 0 LADSPA plugins
*** ERROR: open directory failed '/home/luke/.local/share/SuperCollider/synthdefs'
Cannot use real-time scheduling (RR/5) (1: Operation not permitted)
JackClient::AcquireSelfRealTime error
JackDriver: client name is 'SuperCollider'
SC_AudioDriver: sample rate = 48000.000000, driver's block size = 1024
Exception in World_OpenUDP: unable to bind udp socket

--> Connecting to internal SuperCollider server...
--> Connection established

    _____                 __
   / __  /_  _____  _____/ /_____  ____  ___
  / / / / | / / _ \/ ___/ __/ __ \/ __ \/ _ \
 / /_/ /| |/ /  __/ /  / /_/ /_/ / / / /  __/
 \____/ |___/\___/_/   \__/\____/_/ /_/\___/

   Collaborative Programmable Music. v0.10.3


Hello Luke, just take a moment to pause and focus your creative powers...

nil
user=> (demo (sin-osc 440))
#<synth-node[loading]: user/audition-synth 33>
user=> (demo (sin-osc 440))
#<synth-node[loading]: user/audition-synth 34>
user=> 
``` 

there is no sound coming out - and I have trouble figuring out what is going on. `qjackctl` is installed; however, machine is not rebooted.
I have also MPD running.
Any help would be greatly appreciated!

**Comment by lm-w1n5t0n** [comment]
I had a similar issue where I was getting no sound, but no errors either. It turned out that the SuperCollider server was indeed running, but its outputs were not automatically patched to my system's audio outs. As soon as I patched them manually (using Helvum as a frontend for PipeWire) I started getting sound normally.

Have you tried:
1. Manually launching `qjackctl`
2. Starting the jack server using `qjackctl`
3. Checking the patchbay to see what inputs and outputs are available
4. Starting Overtone
5. Checking the patch bay to see if any new outputs appeared.
6. If they did, patching them to the main output
